proto/caraml/upi/v1/observation_log.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package caraml.upi.v1;
+
+import "caraml/upi/v1/variable.proto";
+import "caraml/upi/v1/table.proto";
```

Comment:
This import is not needed here, right?

Reply:
We need it for `message Value`.

Reply:
I think it's better to use `Variable` for the observation_values field, the `Value` type is tightly coupled with `Table` data type which store its type information.